### DISCUSSION TITLE
GFX100II GPS Not Working

### DISCUSSION THREAD

**Discussion by edcooley** [gkoh/furble#94] [Q&A] [answered]
New user installed on Plus 2 with web install.  I can connect to my GFX100 II and fire the shutter fine.  I am having issues with GPS not recording on exposures.  The GPS option is turned on and GPS data shows date and location.  The option to request smartphone location is enabled on the body and using the Xapp to fire the shutter adds GPS to images.

Any idea what I am doing wrong?

**Comment by gkoh** [comment]
@edcooley I just reinstalled v2.0.0 via web install to my Plus 2 and confirmed GPS works on my X-T30.
Just to confirm, the GPS lock is on? (ie. the circle next to the battery level is solid).

If so, then there's a very real possibility that the GFX might use different GPS traffic and require separate reverse engineering :slightly_frowning_face: 
I don't think anyone has confirmed the GPS works on anything except X-T systems.
Also, just for record, what firmware version are you running on your GFX100II?

**Reply by edcooley** [reply]
Thanks for responding.  The circle next to the battery level is solid.  The GFX100II is at firmware level 2.20.

**Comment by gkoh** [comment]
OK, I think I can reproduce this now, had to leave it overnight so the camera expired everything and now I'm not getting geotag data either despite valid GPS data being received.
This is good, means I should be able to fix it.
Gimme a few days to dig into it.

**Comment by gkoh** [comment, marked as answer]
This was addressed in #95.
